REFERENCE.md

```diff
@@ -0,0 +1,393 @@
+# Reference
```

Comment:
I wouldn't generate this if no parameters have actually be documented using puppet-strings.

Reply:
So, shouldn't rake task take care of this ?

Reply:
which task did you run?

Reply:
this `bundle exec rake strings:generate:reference`

Reply:
oh right.  yeah, that doesn't have any concept of whether the result are going to be worth it.

Reply:
The file is now removed.